didact/camelk/first-integration.md

```diff
@@ -0,0 +1,132 @@
+![Apache Camel](./post-logo-apache-camel-d.png)
+
+# Apache Camel K
+
+Apache Camel K is a lightweight integration framework built from Apache Camel that runs natively on Kubernetes and is specifically designed for serverless and microservice architectures.
+
+Users of Camel K can instantly run integration code written in Camel DSL on their preferred cloud (Kubernetes or OpenShift).
+
+[Check out the Apache Camel K project documentation for more details about the framework.](https://camel.apache.org/camel-k/latest/index.html)
+
+## Apache Camel K in VS Code - Your First Integration
+
+What follows is a simple step-by-step process that helps you create and deploy a Camel K integration on a Minikube instance running locally.
+
+We will:
+
+* Create a folder with a sample Apache Camel integration written in Groovy
```

Comment:
we should not provide a Groovy example as it not in the languages in priority. A java or xml would be better.

Reply:
Fixed